ena-submission/ENA_submission.md

```diff
@@ -310,6 +310,15 @@ When processing is finished the response should look like:
 ]
 ```
 
+# Revising Submissions to ENA
+
+## 1. [Revising Studies (Projects) and Samples](https://ena-docs.readthedocs.io/en/latest/update/metadata/programmatic-study.html)
+Revisions to a study or sample should be submitted the same way as original sequences were submitted, with the `ADD` action in the submission request should be changed to a `MODIFY`. However, the alias must BE THE SAME as the previous version or the assigned accession number must be added for the correct sample/study to be updated. 
+
+## 2. [Revising Assemblies](https://ena-docs.readthedocs.io/en/latest/update/assembly.html)
+It appears that all fields that were explicitly set via the manifest must be updated via an email. This includes changes to any field that is in the `manifest_fields_mapping` field of the default.yaml. Additionally, study and sample reference must stay the same and chromosome names cannot be changed (but new ones can be added). 
```

Comment:
Do you create the email here in his PR automatically, or at least draft the text or is this completely manual right now? Please state what the current state is, what needs to be done manually.